bedrock/mozorg/views.py

```diff
@@ -247,7 +247,7 @@ def get_template_names(self):
         return [template]
 
 
-class ContributeTaskView(TemplateView):
+class ContributeTaskView(l10n_utils.LangFilesMixin, TemplateView):
```

Comment:
I discovered this is required to use conditional l10n tags in the task templates. Without this, the templates throw an error.